dags/postgres_connection_example.py

```diff
@@ -0,0 +1,62 @@
+import boto3
+import sshtunnel
+import psycopg2
+from datetime import datetime
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.python_operator import PythonOperator
```

Comment:
I think this can just be `from airflow.operators.python import PythonOperator`

Reply:
yeah, done.